Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Use the official Python image from the Docker Hub
+FROM python:3.8
+
+# These two environment variables prevent __pycache__/ files.
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+# Make a new directory to put our code in.
+RUN mkdir /code
+
+WORKDIR /code
+
+COPY requirements.txt /code/
+RUN pip install -r requirements.txt
+
+COPY *.py /code/
+COPY config.json /code/
```

Comment:
Baking in the config is not secure. Leave it out and include it via volume

Reply:
I'd totally agree, would prefer to do it in environment variables but didn't want to mess with workflows being new to this project 👍🏻